doc/man1/pbs_server_attributes.7B

```diff
@@ -345,8 +345,9 @@ Default: False
 .br
 Python type: bool
 
-.IP job_requeue_timeout
-Specifies the length of time that can be taken while requeueing a job.
+.IP job_requeue_delay
+Specifies the maximum amount of time the server will take for responding to a job rerun request.
+Once the timeout is exceeded and the job rerun is in progress, same will be reported to the user.
```

Comment:
You may not want to mention "timeout" now. 
suggestion: can this be reworded to something like 
"Amount of time server waits before replying back to client's rerun request."?